src/EditorFeatures/Core/InlineRename/InlineRenameSession.cs

```diff
@@ -863,8 +863,6 @@ private async Task CommitCoreAsync(IUIThreadOperationContext operationContext, b
             if (!_renameInfo.TryOnBeforeGlobalSymbolRenamed(_workspace, changedDocumentIDs, this.ReplacementText))
                 return (NotificationSeverity.Error, EditorFeaturesResources.Rename_operation_was_cancelled_or_is_not_valid);
```

Comment:
Should this also be moved later? I'm not sure why we'd fire this and then do a bunch of background thread work again...?